app/src/test/java/nl/tudelft/b_b_w/model/BlockFactoryUnitTest.java

```diff
@@ -17,8 +17,8 @@
 public class BlockFactoryUnitTest {
 
     private Block _block;
+    final String blockType = "BLOCK";
```

Comment:
and since it is a constant, you have to name the attribute name with capital letters.